src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -529,4 +529,13 @@
   <data name="PreferIsKindFix" xml:space="preserve">
     <value>Use 'IsKind' instead of 'Kind'</value>
   </data>
+  <data name="MissingSourceGeneratorAttributeDescription" xml:space="preserve">
+    <value>'{0}' inherits from ISourceGenerator but does not have the [Generator] attribute</value>
+  </data>
+  <data name="MissingSourceGeneratorAttributeMessage" xml:space="preserve">
+    <value>Missing [Generator] Attribute</value>
+  </data>
+  <data name="MissingSourceGeneratorAttributeTitle" xml:space="preserve">
+    <value>Missing [Generator] Attribute</value>
+  </data>
```

Comment:
nit: since those applies to both languages, I wouldn't use the language-specific syntax in them. Probably just say `GeneratorAttribute` instead of `[Generator] attribute`.

Reply:
Or if you keep the current form, please ensure `attribute` is lower-cased.

Reply:
Updated wording. Going to resolve this, but feel free to open more comments on newer